runtime/src/serde_snapshot/tests.rs

```diff
@@ -219,7 +219,7 @@ fn test_bank_serialize_style(serde_style: SerdeStyle) {
     assert_eq!(dbank.get_balance(&key1.pubkey()), 0);
     assert_eq!(dbank.get_balance(&key2.pubkey()), 10);
     assert_eq!(dbank.get_balance(&key3.pubkey()), 0);
-    bank2.compare_bank(&dbank);
+    assert!(bank2 == dbank);
```

Comment:
nits: how about `assert_eq!`?

Reply:
The issue I ran into with `assert_eq!` was that it requires `Bank` implement `Debug`, which means all the program stack `CowCachedExecutors -> Executor -> InvokeContext` have to all implement Debug , and I didn't want to plumb all of that through 😩

Reply:
@carllin FYI, I've took the initiative here after stressed with not being able to freely `dbg!()` bunch of bank fields: https://github.com/solana-labs/solana/pull/13017/files#diff-b5dead0a38e5f63447595bbeda1428d3aa851d33954343bb1bc38b4b00add5fdR159